README.md

```diff
@@ -0,0 +1,58 @@
+# PLD Generation Tool
+
+The PLD Generation Tool is tool that we made to help us generate good looking pdf
+directly from github issue.
+
+## Install
+
+Before doing anything you have to install `python3`, `docker` and `docker-compose`
+then install the required dependencies by running **install.sh**
+
+```shell
+chmod +x ./install.sh
+./install.sh
+```
+
+## PLD Generation
+
+The PLD Generation is done in multiple steps:
+- Conversion of github issues to Markdown
+- Conversion of Markdown to HTML
+- Conversion of HTML to PDF
+  
+### Convert github issues to Markdown
+
+Simply run the file **mkgen.sh**
+```shell
+chmod +x ./mkgen.sh
+./mkgen.sh
+```
+
+You will next be prompted with a message asking you to enter your Github token.
+
+Enter your github **Token**.
+
+Next you will be asked to select the sprint you want to convert to Markdown.
+Once done the conversion will begin.
+The converted version of the sprint can be found in folder `sprints/{{Sprint Name}}/index.md` and can be edited before converting it to PDF.
+You can also add a file named `sprints/{{Sprint Name}}/bilan.md`, this fill will be used to add a bilan for the sprint.
```

Comment:
You can also add a file named `sprints/{{Sprint Name}}/bilan.md`, this file will be used to add a summary of the sprint.